gitpod-io/website
# fix component alignment on `/contact`

## Description

|Before|After|
|:---:|:---:|
|<img width="1512" alt="image" src="https://github.com/gitpod-io/website/assets/55068936/98e15c1c-4d5c-42a8-aa9b-e363a8b84eb1">|<img width="1497" alt="image" src="https://github.com/gitpod-io/website/assets/55068936/5c3f8155-1417-4d3b-ba9f-fe0180e218a9">|

## Related Issue(s)



Fixes [MKWEB-44](https://linear.app/gitpod/issue/MKWEB-44/weird-alignment-on-contact-page)

## How to test


- Open Preview 
- Go to `/contact` & see 👀
